### DISCUSSION TITLE
Urgent need intersection linkage function intersection linkage

### DISCUSSION THREAD

**Discussion by tgfydzs** [slyh/Cities2-TrafficLightsEnhancement#112] [Ideas]
Modify the road with other mods to remove single left sidewalks. When the road is level, the four intersections can be regarded as a small roundabout or intersection, only need to link the four intersections, the efficiency will be improved!

**Comment by slyh** [comment]
Sorry, I don't understand. Are you building a roundabout using roads instead of the roundabout builder?
